packages/dotlottie-js/src/lottie-animation.ts

```diff
@@ -77,4 +78,51 @@ export class LottieAnimation extends LottieAnimationCommon {
 
     return false;
   }
+
+  /**
+   *
+   * Extract audio assets from the animation.
+   *
+   * @returns boolean - true on error otherwise false on success
+   */
+  protected override async _extractAudioAssets(): Promise<boolean> {
+    if (!this._data) throw new DotLottieError('Asset extraction failed.');
+
+    const animationAssets = this._data.assets as AnimationType['assets'];
+
+    if (!animationAssets) throw new DotLottieError('Asset extraction failed.');
+
+    for (const asset of animationAssets) {
+      if (!('h' in asset) && !('w' in asset) && 'p' in asset && 'e' in asset && 'u' in asset) {
```

Comment:
should we have a `isAudioAsset` for this condition `(!('h' in asset) && !('w' in asset) && 'p' in asset && 'e' in asset && 'u' in asset)` similarly to `isImageAsset` ?